expression/aggregation/agg_to_pb.go

```diff
@@ -99,46 +99,49 @@ func (desc *baseFuncDesc) GetTiPBExpr(tryWindowDesc bool) (tp tipb.ExprType) {
 }
 
 // AggFuncToPBExpr converts aggregate function to pb.
-func AggFuncToPBExpr(sctx sessionctx.Context, client kv.Client, aggFunc *AggFuncDesc) *tipb.Expr {
+func AggFuncToPBExpr(sctx sessionctx.Context, client kv.Client, aggFunc *AggFuncDesc, storeType kv.StoreType) (*tipb.Expr, error) {
 	pc := expression.NewPBConverter(client, sctx.GetSessionVars().StmtCtx)
 	tp := aggFunc.GetTiPBExpr(false)
 	if !client.IsRequestTypeSupported(kv.ReqTypeSelect, int64(tp)) {
-		return nil
+		return nil, errors.New("select request is not supported by client")
```

Comment:
These errors are not reported to the client, because these functions are called in order to check the functions can be pushed down or not. if not, the functions are not pushed down without errors but with some warnings.